chia/cmds/beta.py

```diff
@@ -1,3 +1,5 @@
+# Package: development
```

Comment:
how come this isn't part of the `cmds` project? since it implements click command handlers.

Reply:
The opinion formed (although perhaps not uniformly applied) was that many packages should inherit from a general `cmds` package and then export their own set of commands to be synthesized in the `chia-blockchain` package.